ui/src/features/Editor/components/TopBar/components/ProjectVariablesDialog/VariableEditDialog/components/ChoiceEditor.tsx

```diff
@@ -65,6 +83,20 @@ export const ChoiceEditor: React.FC<Props> = ({ variable, onUpdate }) => {
     setChoiceConfig(getChoiceConfig());
   }, [getChoiceConfig]);
 
+  useEffect(() => {
+    if (assetUrl) {
+      setNewOptionText(assetUrl);
+      clearUrls();
+    }
+  }, [assetUrl, clearUrls]);
+
+  useEffect(() => {
+    if (cmsItemAssetUrl) {
+      setNewOptionText(cmsItemAssetUrl);
+      clearUrls();
+    }
+  }, [cmsItemAssetUrl, clearUrls]);
```

Comment:
We don't need to have urls split since we have no UI to indicate where the url comes from.